### PR TITLE
chore: point thorchain back to shapeshift infra

### PR DESCRIPTION
**Pull request by kaladinlight**
## Description

Point thorchain back to shapeshift infra

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

None

## Testing

- Thorchain and midgard endpoints should still work

### Engineering

:point_up: 

### Operations

:point_up:

### CODE REVIEW

File: .env.develop [hunk 1]
```diff
@@ -44,7 +44,7 @@ REACT_APP_POLYGON_NODE_URL=https://dev-daemon.polygon.shapeshift.com
 REACT_APP_GNOSIS_NODE_URL=https://dev-daemon.gnosis.shapeshift.com
 REACT_APP_ARBITRUM_NODE_URL=https://dev-daemon.arbitrum.shapeshift.com
 REACT_APP_COSMOS_NODE_URL=https://dev-daemon.cosmos.shapeshift.com
-REACT_APP_THORCHAIN_NODE_URL=https://dev-daemon.thorchain.shapeshift.com/lcd
```
Comment (0xApotheosis) [inline .env.develop:47]: We probably want to keep `/lcd` as a part of the env var so we don't have to plumb it through everywhere on change.

Comment (0xApotheosis) [reply .env.develop:47]: See this PR, which intentionally did this: https://github.com/shapeshift/web/pull/5485

Comment (kaladinlight) [reply .env.develop:47]: For whatever reason, the csp rules weren't behaving well with the path so I punted and switched back. On the plus side, we have access to` /rpc` if we ever need without having to add a new env var I suppose..

Comment (0xdef1cafe) [reply .env.develop:47]: as discussed if this is causing csp issues let's leave as is

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**